src/peep/peep.c

```diff
@@ -556,13 +556,13 @@ static void sub_68F41A(rct_peep *peep, int index)
 
 					switch (chosen_thought){
 					case PEEP_THOUGHT_TYPE_HUNGRY:
-						peep_head_for_nearest_ride_with_flags(peep, 0x00800000);
+						peep_head_for_nearest_ride_with_flags(peep, RIDE_TYPE_FLAG_SELLS_FOOD);
 						break;
 					case PEEP_THOUGHT_TYPE_THIRSTY:
-						peep_head_for_nearest_ride_with_flags(peep, 0x01000000);
+						peep_head_for_nearest_ride_with_flags(peep, RIDE_TYPE_FLAG_SELLS_DRINKS);
 						break;
 					case PEEP_THOUGHT_TYPE_BATHROOM:
-						peep_head_for_nearest_ride_with_flags(peep, 0x00200000);
+						peep_head_for_nearest_ride_with_flags(peep, RIDE_TYPE_FLAG_PEEP_SHOULD_GO_INSIDE_FACILITY);
```

Comment:
I think i made a mistake here it should have been `0x02000000` i.e. RIDE_TYPE_FLAG_IS_BATHROOM